### ISSUE TITLE
Bootstrap Version

### ISSUE THREAD

**Issue by SachaG** [Meteor-Alt/accounts#5]
Have you thought about having a bootstrap-compatible version, maybe using https://react-bootstrap.github.io ? 

I don't think you'd even need to change the styling, just make sure the markup outputted matches the one expected by Bootstrap?

**Comment by emmostrom** [comment]
I like Bootstrap and use it a lot and I'm not opposed to creating an example project that shows how to customize the accounts package - but I don't want to try and keep up with maintaining a bunch of different styled versions (Bootstrap, Material, ...)  I would hope others would create their own packages that extend react-accounts-unstyled.

Not sure I agree with the react-bootstrap project's philosophy.  It seems to me the problem they are trying to solve is because Bootstrap isn't being used correctly.

**Comment by zeroasterisk** [comment]
I agree, I'd love to see a bootstrap styled example... you only have to go so far as templating out one little bit, and I could style the rest.
